src/Microsoft.AspNetCore.Mvc.TagHelpers/ValidationMessageTagHelper.cs

```diff
@@ -71,37 +71,29 @@ public ValidationMessageTagHelper(IHtmlGenerator generator)
                     };
                 }
 
+                string message = null;
+                TagHelperContent tagHelperContent = null;
```

Comment:
No need for this outside the next block i.e. just use `var tagHelperContent = await output.GetChildContentAsync();`